src/main/kotlin/no/nav/amt/distribusjon/journalforing/model/Endringshendelse.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.amt.distribusjon.journalforing.model
+
+import no.nav.amt.distribusjon.hendelse.model.Hendelse
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class Endringshendelse(
```

Comment:
Jeg skjønner at det er ikke enkelt å finne et godt navn for denne. Men jeg måtte tenke og lese litt mye før jeg forstår hvorfor dette er en `Endringshendelse` og hvorfor det ikke bare er en `Hendelse`. Kan det gi mer mening at vi kaller det for `Endringsvedtak` selv om det potensielt ikke er et journalført vedtak enda?

Reply:
Ja, det kan vi godt :)